Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/references/EditExternalReferencePanel.java

```diff
@@ -160,11 +160,10 @@ private void populateExternalNames() {
 		extLibName.clearModel();
 		extLibName.addItem(Library.UNKNOWN);
 		Arrays.sort(names);
-		for (int i = 0; i < names.length; i++) {
-			if (Library.UNKNOWN.equals(extLibName)) {
-				continue;
+		if (!Library.UNKNOWN.equals(extLibName)) {
```

Comment:
this still isn't correct.  The original code was trying to compare Library.UNKNOWN (ie. ```"<EXTERNAL>"```) to the current element from names.

```java
		for (String name : names) {
			if (Library.UNKNOWN.equals(name)) {
				continue;
			}
			extLibName.addItem(name);
		}
```

Reply:
That was not what the original code did. At all.

Reply:
Fixed though

Reply:
Yeah, it wasn't doing it, but I could tell that was what it was trying to do. :crossed_fingers: